docs/guides/single-fetch.md

```diff
@@ -304,9 +290,7 @@ Each `loader`/`action` receives its own unique `response` instance so you cannot
   - `headers.append` can be used to set the same header from both a parent and child handler
   - `headers.delete` can be used to delete a value set by a parent handler, but not a value set from a child handler
 
-Because Single Fetch supports naked object returns, and you no longer need to return a `Response` instance to set status/headers, the `json`/`redirect`/`redirectDocument`/`defer` utilities can be considered deprecated when using Single Fetch.
-
-**\*However**, this does not mean you need to change them all right away in order to adopt Single Fetch!\*
+Because Single Fetch supports naked object returns, and you no longer need to return a `Response` instance to set status/headers, the `json`/`redirect`/`redirectDocument`/`defer` utilities should be considered deprecated when using Single Fetch.
```

Comment:
Maybe append this with something super clear like "these will remain for the duration of v2 so you don't need to remove the immediately.  They will likely be removed in the next major version so we recommend remove them incrementally between now and then." or something

Reply:
Added! Thanks for the suggestion